Changelog.md

```diff
@@ -1,5 +1,88 @@
 # Changelog
 
+### 0.13.0
+
+**Image:**  `quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.13.0`
+
+*New Features:*
+
+- NGINX 1.13.12
+- Support for gRPC:
+  - The annotation `nginx.ingress.kubernetes.io/grpc-backend: "true"` enable this feature
+  - If the gRPC service requires TLS `nginx.ingress.kubernetes.io/secure-backends: "true"`
+- Configurable load balancing with EWMA
+- Add NoAuthLocations and default it to "/.well-known/acme-challenge"
```

Comment:
Should `lua-resty-waf` be included as a newly supported feature as well? https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/annotations.md#lua-resty-waf

Also session affinity is supported in dynamic mode.

Reply:
Good point. Done.